src/librustc/ty/context.rs

```diff
@@ -1106,11 +1106,11 @@ impl<'a, 'gcx, 'tcx> TyCtxt<'a, 'gcx, 'tcx> {
         self.global_arenas.generics.alloc(generics)
     }
 
-    pub fn alloc_steal_mir(self, mir: Mir<'gcx>) -> &'gcx Steal<Mir<'gcx>> {
+    pub fn alloc_steal_mir(self, mir: Body<'gcx>) -> &'gcx Steal<Body<'gcx>> {
         self.global_arenas.steal_mir.alloc(Steal::new(mir))
     }
 
-    pub fn alloc_mir(self, mir: Mir<'gcx>) -> &'gcx Mir<'gcx> {
+    pub fn alloc_mir(self, mir: Body<'gcx>) -> &'gcx Body<'gcx> {
```

Comment:
These methods should probably also end in `_mir_body` (not sure if we should do this in this PR though).

Reply:
Okay... We should wait for @oli answer, right?

Reply:
you're pinging the wrong oli ;) I'm @oli-obk 

yea, that rename makes sense, but let's do it in a separate PR to reduce bitrot